DistFiles/factoryCollections/Templates/Wall Calendar/configure.ts

```diff
@@ -43,11 +43,11 @@ class CalendarConfigurator {
     //
     updateDom():void {
         var year:string = this.configObject.year;
-        var originalMonthsPicturePage:HTMLDivElement = $('.calendarMonthTop')[0];
-        var pageToInsertAfter:HTMLDivElement = originalMonthsPicturePage;
+        var originalMonthsPicturePage:HTMLElement = $('.calendarMonthTop')[0];
+        var pageToInsertAfter:HTMLElement = originalMonthsPicturePage;
         for (var month:number = 0; month < 12; month++) {
 
-            var monthsPicturePage:HTMLDivElement = $(originalMonthsPicturePage).clone();
+            var monthsPicturePage:HTMLElement = $(originalMonthsPicturePage).clone()[0];
```

Comment:
Clone() returned a JQuery object, not an HTMLElement (or HTMLDivElement). In order to fix the error messages here caused by type-checking, I had to specify the internal HTMLElement in the JQuery object. It was an interesting 'feature' of JQuery that it actually worked before! Fortunately, the jquery object always contained one and only one item, which was the page we wanted.